### PR TITLE
Switch clusterinfo to print recommended proxy endpoints

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

File: pkg/kubectl/cmd/clusterinfo.go [hunk 1]
```diff
@@ -102,9 +102,9 @@ func RunClusterInfo(f cmdutil.Factory, out io.Writer, cmd *cobra.Command) error
 				}
 			} else {
 				if len(client.GroupVersion.Group) == 0 {
-					link = client.Host + "/api/" + client.GroupVersion.Version + "/proxy/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name
+					link = client.Host + "/api/" + client.GroupVersion.Version + "/namespaces/" + service.ObjectMeta.Namespace + "/services/" + service.ObjectMeta.Name + "/proxy"
```
Comment (soltysh) [inline pkg/kubectl/cmd/clusterinfo.go]: Do we have any tests covering this?

**Review by soltysh — COMMENTED** [review]
A question, but LGTM.

**Comment by liggitt** [comment]
@k8s-bot kops aws e2e test this

**Pull request merged**